### PR TITLE
Merged last fixes from Code-11-SP3

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

**Comment by jreidinger** [comment]
if you test it, I am fine with it as I am not such strict on ycp :)

**Comment by mchf** [comment]
It is tested in SP3. I don't have working SP4 image, however I don't expect any issue there anyway.

File: yast2.spec.in [hunk 1]
```diff
@@ -42,7 +42,8 @@ Conflicts:      yast2-inetd <= 2.13.4
 Conflicts:      yast2-tune < 2.15.6
 Obsoletes:      yast2-mail-aliases <= 2.14.0
 Conflicts:      yast2-storage < 2.16.4
-Conflicts:      yast2-network < 2.16.6
+# older yast2-network uses old device type detection API
```
Comment (mvidner) [inline yast2.spec.in]: 2.17.201

**Comment by mvidner** [comment]
Also here, Please fix the dependency version, then it LGTM.

**Comment by mchf** [comment]
Dependency fixed. Merging. Thanks for reviews

**Pull request merged**